conda-store-server/conda_store_server/action/generate_constructor_installer.py

```diff
@@ -0,0 +1,108 @@
+import os
+import pathlib
+import subprocess
+import sys
+import tempfile
+
+import yaml
+from conda_store_server import action, schema
+
+
+@action.action
+def action_generate_constructor_installer(
+    context,
+    conda_command: str,
+    specification: schema.CondaSpecification,
+    installer_dir: pathlib.Path,
+    version: str,
+):
+    # Helpers
+    def print_cmd(cmd):
+        context.log.info(f"Running command: {' '.join(cmd)}")
+        context.log.info(
+            subprocess.check_output(cmd, stderr=subprocess.STDOUT, encoding="utf-8")
+        )
```

Comment:
Same helper as in `conda-store-server/conda_store_server/action/generate_lockfile.py`

Reply:
If this helper already exists somewhere else, is it worth refactoring it to a utils module? I'm not a big fan of the `print_cmd` name because it doesn't imply execution. Maybe something `logged_subprocess` or something. I'd also forward **kwargs to the subprocess call in case we want to add options (e.g. timeouts; see below).